src/cli/auth.js

```diff
@@ -0,0 +1,119 @@
+import fs from 'fs'
+import path from 'path'
+import prompt from 'prompt'
+import chalk from 'chalk'
+import validUrl from 'valid-url'
+import axios from 'axios'
+import Confirm from 'prompt-confirm'
+
+import util from '../util'
+
+const AUTH_FILE = '.auth.json'
+
+// TODO: this part replicates `start.js`, refactor
+const getDataDir = () => {
+  const projectPath = path.resolve('.')
+
+  const botfile = path.join(projectPath, 'botfile.js')
+  if (!fs.existsSync(botfile)) {
+    util.print('error', `(fatal) No ${chalk.bold('botfile.js')} file found at: ` + botfile)
+    process.exit(1)
+  }
+
+  // eslint-disable-next-line no-eval
+  const bf = eval('require')(botfile)
```

Comment:
@emirotin , I saw comment above that this needs refactoring, but what could be a reason for using `eval('require')`? :hushed:

Reply:
It's done so everywhere with a comment that this is needed for webpack keeping dynamic importing (not resolving and including the modules in the bundle).
I will eventually check if there's a more elegant way to do so but for now will stick to this style

Reply:
Yeah it's the only work around I found for webpack to not bundle these. I guess we could also have a special loader `eval('skip!' + botfile)`